src/functions/New-PuppetDscModule.ps1

```diff
@@ -84,6 +84,11 @@ Function New-PuppetDscModule {
       If (Test-SymLinkedItem -Path $OutputDirectory -Recurse) {
         Stop-PsfFunction -EnableException $true -Message "The specified output folder '$OutputDirectory' has a symlink in the path; CIM class parsing will not work in a symlinked folder, specify another path"
       }
+      Try {
+        $null = Test-WSMan -ErrorAction Stop
+      } Catch {
+        Stop-PsfFunction -EnableException $true -Message "PSRemoting does not appear to be enabled; in order to parse CIM instances, the function needs to do a stubbed DSC invocation; this will fail without PSRemoting enabled. Enable PSRemoting (possibly via the Enable-PSRemoting command) before retrying."
```

Comment:
Any possibility to also surface the underlying error, so folks can understand their situation in edge-cases we didn't consider?

Reply:
Good call, added that in!